azure-pipelines.yml

```diff
@@ -55,8 +53,13 @@ jobs:
   - script: python -m pip install --upgrade pip && python -m pip install tox
     displayName: 'Install tox'
 
-  - script: |
-      call scripts/setup-coverage-vars.bat || goto :eof
+  - bash: |
+      if [[ "$PYTEST_COVERAGE" == "1" ]]; then
```

Comment:
This should be calling `scripts/report-coverage.sh` no?

Reply:
No, that's the env setup.

Reply:
This _could_ be merged into tox.ini, and then we would end up using TOXENV=…-coverage basically.

Reply:
Oh duh sorry